index.js

```diff
@@ -4,20 +4,22 @@ const create = prod => (start, sink) => {
     sink(0, () => {});
     return;
   }
-  let started = disposed = got1 = got2 = false;
-  sink(0, t => {
-    got1 = got1 || t === 1;
```

Comment:
im not sure what was the purpose of those 2, especially `got2` seemed off, if sink sends 2 upwards I dont see why it should be resend downwards

I've removed both, tests still pass 😉 If it broke some untested case, please let me know